lang/cs/Org.Apache.REEF.Bridge.Core.Tests/Fail/Task/Driver.cs

```diff
@@ -0,0 +1,177 @@
+
+                ConfigurationModule taskConfig =
+                    TaskConfiguration.ConfigurationModule.Set(TaskConfiguration.Identifier, _taskId);
+
+                switch (_failTaskName)
+                {
+                    case "FailTask":
+                        taskConfig = taskConfig.Set(TaskConfiguration.Task, GenericType<FailTask>.Class);
+                        break;
+                    case "FailTaskCall":
+                        taskConfig = taskConfig.Set(TaskConfiguration.Task, GenericType<FailTaskCall>.Class);
+                        break;
+                    case "FailTaskMsg":
+                        taskConfig = taskConfig
+                            .Set(TaskConfiguration.Task, GenericType<FailTaskMsg>.Class)
+                            .Set(TaskConfiguration.OnMessage, GenericType<FailTaskMsg>.Class);
+                        break;
+                    case "FailTaskSuspend":
+                        taskConfig = taskConfig
+                            .Set(TaskConfiguration.Task, GenericType<FailTaskSuspend>.Class)
+                            .Set(TaskConfiguration.OnSuspend, GenericType<FailTaskSuspend>.Class);
+                        break;
+                    case "FailTaskStart":
+                        taskConfig = taskConfig
+                            .Set(TaskConfiguration.Task, GenericType<FailTaskStart>.Class)
+                            .Set(TaskConfiguration.OnTaskStart, GenericType<FailTaskStart>.Class);
+                        break;
+                    case "FailTaskStop":
+                        taskConfig = taskConfig
+                            .Set(TaskConfiguration.Task, GenericType<FailTaskStop>.Class)
+                            .Set(TaskConfiguration.OnTaskStop, GenericType<FailTaskStop>.Class)
+                            .Set(TaskConfiguration.OnClose, GenericType<FailTaskStop>.Class);
+                        break;
+                    case "FailTaskClose":
+                        taskConfig = taskConfig
+                            .Set(TaskConfiguration.Task, GenericType<FailTaskClose>.Class)
+                            .Set(TaskConfiguration.OnClose, GenericType<FailTaskClose>.Class);
+                        break;
+                    default:
+                        break;
+                }
+
+                eval.SubmitContextAndTask(contextConfig, taskConfig.Build());
+            }
+            catch (BindException ex)
+            {
+                Log.Log(Level.Warning, "Configuration error", ex);
+                throw new DriverSideFailure("Configuration error", ex);
+            }
+        }
+
+        public void OnNext(IRunningTask task)
+        {
+            Log.Log(Level.Info,
+                "TaskRuntime: {0} expect {1}",
+                new Object[] { task.Id, _taskId });
+
+            if (!_taskId.Equals(task.Id))
+            {
+                throw new DriverSideFailure("Task ID " + task.Id +
```

Comment:
BTW, C# supports string interpolation where you can embed the variable into the string by specifying the $ in front of the string. This could be instead $"Task ID{ {task.Id} not equal expected ID {taskId}"